desktop/cmp/rest/RestGridModel.js

```diff
@@ -174,60 +192,71 @@ export class RestGridModel {
         this.formModel.openClone(clone);
     }
 
-    @action
-    deleteSelection() {
```

Comment:
As mentioned in Changelog, I did not find uses of this or editSelection. Assuming that RestGrids aren't used too extensively in client app, so while this hypothetically could be a breaking change, more likely it's just cruft